openlibrary/templates/covers/book_cover.html

```diff
@@ -15,7 +15,7 @@
     <div class="coverMagic cover-animation">
             <div class="SRPCover bookCover" style="display: $cond(cover_url, 'block', 'none');">
                 <a href="$cover_lg" aria-controls="seeImage"
-                    class="coverLook dialog--open" title="Pull up a bigger book cover"><img itemprop="image" src="$cover_url" srcset="$cover_lg 2x" class="cover" alt="Cover of: $title | $author_names"/></a>
+                    class="coverLook dialog--open" title="$_('Pull up a bigger book cover')"><img itemprop="image" src="$cover_url" srcset="$cover_lg 2x" class="cover" alt="$_('Cover of: %(title)s', title=title) $_('by') $author_names"/></a>
```

Comment:
This is better grouped into one string; some languages might need these to be in a reversed order.

```suggestion
                    class="coverLook dialog--open" title="$_('Pull up a bigger book cover')"><img itemprop="image" src="$cover_url" srcset="$cover_lg 2x" class="cover" alt="$_('Cover of: %(title)s by $(authors)s', title=title, authors=author_names)"/></a>
```